python/hail/methods/statgen.py

```diff
@@ -428,16 +427,16 @@ def logreg(dataset, test, y, x, covariates=[], root='logreg'):
     convergence issues, Hail also emits three variant annotations which
     summarize the iterative fitting process:
 
-    ================ ======================= ======= ===============================
-    Test             Field                   Type    Value
-    ================ ======================= ======= ===============================
-    Wald, LRT, Firth `logreg.fit.n_iter`     int32   number of iterations until
-                                                     convergence, explosion, or
-                                                     reaching the max (25 for
-                                                     Wald, LRT; 100 for Firth)
-    Wald, LRT, Firth `logreg.fit.converged`  bool    ``True`` if iteration converged
-    Wald, LRT, Firth `logreg.fit.exploded`   bool    ``True`` if iteration exploded
-    ================ ======================= ======= ===============================
+    ================ ========================= ======= ===============================
+    Test             Field                     Type    Value
+    ================ ========================= ======= ===============================
+    Wald, LRT, Firth `logreg.fit.n_iterations` int32   number of iterations until
```

Comment:
Pythonic: num_iterations

Reply:
We mix this all over our interface, but I think I do prefer `num`. I'll change it.

Reply:
actually, I'm not so sure. Let's poll the group

Reply:
did some research, fine to change to num. We'll need to change the sample/variant qc annotations too